Source/WebCore/platform/graphics/cg/ImageUtilitiesCG.cpp

```diff
@@ -255,41 +261,53 @@ static RefPtr<SharedBuffer> expandSVGImageToData(SVGImage& svgImage, ASCIILitera
     return SharedBuffer::create(destinationData.get());
 }
 
-RefPtr<SharedBuffer> createIconDataFromImageData(std::span<const uint8_t> data, std::span<const unsigned> lengths)
+void createIconDataFromImageData(std::span<const uint8_t> data, const Vector<unsigned>& lengths, CompletionHandler<void(RefPtr<SharedBuffer>&&)>&& completionHandler)
 {
     // Supported ICO image sizes by ImageIO.
-    constexpr std::array<unsigned, 5> availableLengths { { 16, 32, 48, 128, 256 } };
+    auto targetLengths = lengths.isEmpty() ? Vector<unsigned>({ 16, 32, 48, 128, 256 }) : lengths;
```

Comment:
I don’t think we should copy this into a `Vector` until we have to call `SVGImage::tryCreateFromDataAndWaitForSubresources`. We should go back to using `std::span` and `std::array` and then make the vector with `targetLengthVector = Vector<unsigned> { targetLengths }`  instead of `targetLengths = WTFMove(targetLengths)`.